### PR TITLE
Feature/cfn sdk lookups

### PR DESCRIPTION
**Pull request by mikegwhit**
Adds Cfn SDK functions and provides simple error message.

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (codeflyer) [inline config/default.json:1]: Not sure but this file should't be added to the repo

File: src/sdk/cloudformation.ts [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+// export const getInProgressStacks = (): void => {
+
+// }
+
+/**
+ * Gets the resources included in a stack.
+ */
+export const getStackResources = async (StackName: string): Promise<LooseObject> => {
+  const cfn = await getCfn()
+  return new Promise((resolve, reject) => {
```
Comment (codeflyer) [inline src/sdk/cloudformation.ts:45]: All the SDK method can used directly using the promise adding `.promise()`  and skip the callback param

(EG.)
```
      listResult = await s3
        .listObjectsV2({
          Bucket: config.trail.bucketName,
          Prefix: `${prefix}/${config.region}/${day}`,
          ContinuationToken: listResult
            ? listResult.NextContinuationToken
            : undefined
        })
        .promise()
```

**Review by codeflyer — COMMENTED** [review]
LGTM, just a copule of comments

File: src/sdk/cloudformation.ts (hunk 2 above)
Comment (mikegwhit) [reply src/sdk/cloudformation.ts]: Done

**Review by codeflyer — APPROVED** [review]
LGTM

**Pull request merged**